doc/api/url.md

```diff
@@ -75,14 +70,19 @@ const myURL =
 ```
 
 ## The WHATWG URL API
+
+### Class: URL
 <!-- YAML
 added: v7.0.0
+changes:
+  - version: REPLACEME
+    pr-url: REPLACEME
```

Comment:
we should be able to go ahead and set this, no?

Reply:
thanks. updated